eZ/Bundle/EzPublishRestBundle/Resources/config/value_object_visitors.yml

```diff
@@ -159,35 +159,35 @@ services:
     ezpublish_rest.output.value_object_visitor.InvalidArgumentException:
         parent: ezpublish_rest.output.value_object_visitor.base
         class: %ezpublish_rest.output.value_object_visitor.InvalidArgumentException.class%
-        arguments: [ %kernel.debug% ]
+        arguments: [ %kernel.debug%, @translator ]
```

Comment:
I considered this, but the same goes for `%kernel.debug%`
I can do that, but I'll then refactor with setters if that's OK with you.